index.html

```diff
@@ -72,5 +76,208 @@ <h2 class="introduction__heading--2">LET’S CONNECT</h2>
         </div>
       </section>
     </main>
+    <!-- WORKS SECTION -->
+    <section class="works">
+      <div class="works__tonic-mps">
+        <img
+          class="works__image"
+          src="/resources/img/project1.png"
+          alt="Project Image"
+        />
+        <h3 class="works__heading">Tonic</h3>
+        <ul class="works__list--1">
+          <li class="canopy">Canopy</li>
+          <li><span class="dot"></span></li>
+          <li>Back End Dev</li>
+          <li><span class="dot"></span></li>
+          <li>2015</li>
+        </ul>
+        <p class="works__paragraph">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works__list--2">
+          <li class="works__list--2-item">html</li>
+          <li class="works__list--2-item">css</li>
+          <li class="works__list--2-item">javaScript</li>
+        </ul>
+        <div class="btn">
+          <a class="btn-primary works__btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="works__tonic-mps">
```

Comment:
- [ ] [REQUIRED] Please, use a semantic tag instead of `div` as requested in project requirements. You can use `<article>`.

Reply:
@OmarRamoun Thank you so much for your kind review! I made the changes requested and noticed I was missing the border radius for the articles, thank you for your observations!